language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaSyntaxApiQuoteRequest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.extensions.ballerina.document;
+
+import org.eclipse.lsp4j.TextDocumentIdentifier;
+
+/**
+ * Represents a request for a Ballerina Syntax API Quote.
+ *
+ * @since 2.0.0
+ */
+public class BallerinaSyntaxApiQuoteRequest {
```

Comment:
We don't have to use `Ballerina` prefix in all the classes. Can we clean up these class names?